src/pages/api/request-token.ts

```diff
@@ -20,7 +20,7 @@ export type ApiRequestTokenResponse = {
   hash?: string
 }
 
-const VERIFIER = 'https://hcaptcha.com/siteverify'
+const VERIFIER = 'https://www.google.com/recaptcha/api/siteverify'
 const BURN_AMOUNT = 0.3 * 10 ** 10
```

Comment:
How many messages user can do with the 0.3 energy? Maybe make sense increase like 50 messages per one captcha

Reply:
Changed it to 0.5, each tx costs about ~0.09 * 10 ** 10
so it will be enough for more than 50 txs